### PR TITLE
Rearrange setup.md and Add CLI Documentation

### PR DESCRIPTION
**Pull request by amalpaul54111**
1. Add documentation for the CLI
2. Copy essential parts of the CLI documentation to the setup.md

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -11,6 +11,83 @@ You should have Docker Engine and Docker Compose installed. This is very fast an
...
+## Recommended: CLI Installation
+
+### Installation
+* Then, run the following line to install the Canasta CLI:
+
+```
+curl -fsL https://raw.githubusercontent.com/CanastaWiki/Canasta-CLI/main/install.sh | bash
+``` 
+
+### All available commands
```
Comment (jeffw16) [inline setup.md:23]: Remove this subsection

File: setup.md [hunk 2]
```diff
@@ -40,7 +117,7 @@ You should have Docker Engine and Docker Compose installed. This is very fast an
 ### After installation
 There's several things you can do to polish up your wiki so it's ready for use:
 
-* To add popular extensions quickly, visit the [extensions setup](extensions-setup) page to explore your choices.
+* To add popular extensions quickly, visit the [canasta extensions](#enabling-extensions) page to explore your choices.
```
Comment (jeffw16) [inline setup.md:120]: Capitalize Canasta

**Review by jeffw16 — CHANGES_REQUESTED** [review]

File: setup.md [hunk 3]
```diff
@@ -11,6 +11,54 @@ You should have Docker Engine and Docker Compose installed. This is very fast an
 * [CentOS](https://docs.docker.com/engine/install/centos/)
 * More available at [https://docs.docker.com/engine/install/](https://docs.docker.com/engine/install/)
 
+## Recommended: CLI Installation
```
Comment (jeffw16) [inline setup.md:14]: @yaronkoren says: Need to add what the CLI is and why it's recommended. One or two sentences.

**Review by jeffw16 — CHANGES_REQUESTED** [review]

File: setup.md (hunk 3 above)
Comment (amalpaul54111) [reply setup.md:14]: Yeah, sure.

**Pull request merged**